ceph/ceph_admin/bootstrap.py

```diff
@@ -185,6 +185,7 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
                     mon-ip: <node-name>
         """
         self.cluster.setup_ssh_keys()
+        skip_cephadm_bootstrap = config.get("skip_cephadm_bootstrap", False)
```

Comment:
IMO, we could decouple a couple of methods or create a new module and then transition into it. Adding this flag wouldn't be the right step working towards the goal.